src/codeu/chat/util/MessageJson.java

```diff
@@ -0,0 +1,19 @@
+package codeu.chat.util;
+
+import codeu.chat.util.Uuid;
+import codeu.chat.common.Message;
+
+public class MessageJson extends TransactionJson {
+  public String author;
+  public String content;
+  public String conversation;
+
+  public MessageJson(String action, Uuid conversation, Message message) {
+    super.action = action;
```

Comment:
Nit: create a constructor for TransactionJson(action, uuid, creation) and call super(action, message.id.toString, creation.toString()